torch_glow/src/PyTorchCommon.cpp

```diff
@@ -227,8 +247,15 @@ at::Tensor glowTypeToEmptyPTTensor(const glow::Type &glowType) {
 }
 
 glow::Tensor ptTensorToGlowTensor(const at::Tensor &ptTensor) {
-  auto glowType = ptTypeToGlowType(*c10::TensorType::create(ptTensor));
-  return glow::Tensor(ptTensor.data_ptr(), &glowType);
+  auto ptType = *c10::TensorType::create(ptTensor);
+  if (ptTensor.is_quantized()) {
```

Comment:
Is this path being tested anywhere? If not let's make sure we make a test for it.

Reply:
I can do it by using torch.nn.quantized.Conv2d, however that might introduce the error of accuracy.